client/packages/requisitions/src/utils.ts

```diff
@@ -67,7 +67,11 @@ export const isRequestDisabled = (request: RequestRowFragment): boolean => {
 };
 
 export const isResponseDisabled = (request: RequestRowFragment): boolean => {
-  return request.status !== RequisitionNodeStatus.New;
+  return (
```

Comment:
I was confused by the use of Request in this function. Would you mind fixing a previous copy paste error?
```
export const isResponseDisabled = (response: ResponseRowFragment): boolean => {
  return (
    response.status !== RequisitionNodeStatus.New ||
    response.approvalStatus === RequisitionNodeApprovalStatus.Pending ||
    response.approvalStatus === RequisitionNodeApprovalStatus.Denied
  );
};
```